### DISCUSSION TITLE
looking for freecad files

### DISCUSSION THREAD

**Discussion by kinghat** [Stemfie3D/STEMFIE-file-repository#6] [General]
i was looking around the repo for the freecad file for your parts. i was only able to find the one in templates. the reason im looking is i would like to see your freecad modeling flow. i would also like to understand how you organize and version a project like this.

**Comment by Mr3dRobots** [comment]
> je cherchais dans le dépôt le fichier freecad pour vos pièces. je n'ai pu trouver celui-ci que dans les modèles. la raison pour laquelle je cherche est que j'aimerais voir votre flux de modélisation freecad. j'aimerais également comprendre comment vous organisez et versionnez un projet comme celui-ci.

Hello, I suggest you come to the Telegram group so that you can talk with the entire community about the project and have more information on it.

telegram channel link: https://t.me/Stemfie3DGroup

freecad template download link: https://get.stemfie.org/Templates%20-%20FreeCAD/

video in freecad showing the approach for a simple piece: https://www.youtube.com/watch?v=l6JwDDBP3M8

**Comment by Stemfie3D** [comment]
For now, only the template file is available as a FreeCAD file. In the future, more files will be released in FreeCAD format.
